app/org/sagebionetworks/bridge/models/notifications/NotificationRegistration.java

```diff
@@ -0,0 +1,76 @@
+package org.sagebionetworks.bridge.models.notifications;
+
+import org.sagebionetworks.bridge.dynamodb.DynamoNotificationRegistration;
+import org.sagebionetworks.bridge.models.BridgeEntity;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+@JsonDeserialize(as=DynamoNotificationRegistration.class)
+public interface NotificationRegistration extends BridgeEntity {
+
+    public static NotificationRegistration create() {
+        return new DynamoNotificationRegistration();
+    }
+    
+    /**
+     * The healthCode of the user registering their device for notifications.
+     */
+    public String getHealthCode();
+    
+    /** @see #getHealthCode */
+    public void setHealthCode(String healthCode);
+
+    /**
+     * The GUID of this registration that is returned to the client to update the 
+     * device identifier in subsequent requests. Can also be used to unregister 
+     * for notifications. 
+     */
+    public String getGuid();
+    
+    /** @see getGuid */
+    public void setGuid(String guid);
+    
+    /**
+     * The platformEndpoint ARN generated when the user's device is registered 
```

Comment:
What's the difference between platformARN, endpointARN, and platformEndpoint ARN?